src/app/store/onboarding/onboarding.slice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import {
+  OnboardingSteps,
+  OnboardingStepsStatus,
+  OnboardingStepStatus,
+} from '@shared/models/onboarding-types';
+
+interface OnboardingState {
+  hideSteps: boolean;
+  stepsStatus: OnboardingStepsStatus;
+}
+
+const initialState: OnboardingState = {
+  hideSteps: false,
+  stepsStatus: {
+    [OnboardingSteps.BackUpSecretKey]: OnboardingStepStatus.Done,
+    [OnboardingSteps.AddFunds]: OnboardingStepStatus.Start,
+    [OnboardingSteps.RegisterName]: OnboardingStepStatus.Start,
+    [OnboardingSteps.ExploreApps]: OnboardingStepStatus.Start,
+  },
+};
+
+export const onboardingSlice = createSlice({
+  name: 'onboarding',
+  initialState,
+  reducers: {
+    setHideSteps(state, action: PayloadAction<boolean>) {
+      state.hideSteps = action.payload;
+    },
+    setStepsStatus(state, action: PayloadAction<OnboardingStepsStatus>) {
+      state.stepsStatus = action.payload;
+    },
```

Comment:
These actions are okay, but where possible to try avoid "setter" actions https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters 

Sometimes this could just be a name change `hideOnboardingSteps`/`showOnboardingSteps`.